monai/apps/reconstruction/my_mri_dictionary.py

```diff
@@ -0,0 +1,272 @@
+from monai.transforms.intensity.array import NormalizeIntensity
+from monai.transforms.transform import MapTransform
+from monai.utils.type_conversion import convert_to_tensor
+
+
+class RandomKspaceMaskd(MapTransform):
+    """
+    Dictionary-based wrapper of :py:class:`monai.transforms.RandomKspaceMask`.
+
+    Args:
+        keys: keys of the corresponding items to be transformed.
+            See also: monai.transforms.MapTransform
+        center_fractions: Fraction of low-frequency columns to be retained.
+            If multiple values are provided, then one of these numbers is
+            chosen uniformly each time.
+        accelerations: Amount of under-sampling. This should have the
+            same length as center_fractions. If multiple values are provided,
+            then one of these is chosen uniformly each time.
+        spatial_dims: Number of spatial dims (e.g., it's 2 for a 2D data; it's
+            also 2 for psuedo-3D datasets like the fastMRI dataset).
+            The last spatial dim is selected for sampling. For the fastMRI
+            dataset, k-space has the form (...,num_slices,num_coils,H,W)
+            and sampling is done along W. For a general 3D data with the
+            shape (...,num_coils,H,W,D), sampling is done along D.
+        is_complex: if True, then the last dimension will be reserved
+            for real/imaginary parts.
+        seed: Seed for the random number generator. Setting the seed
+            ensures the same mask is generated each time for the same shape.
+        allow_missing_keys: don't raise exception if key is missing.
+    """
+
+    backend = RandomKspaceMask.backend
+
+    def __init__(
+        self,
+        keys: KeysCollection,
+        center_fractions: Sequence[float],
+        accelerations: Sequence[float],
+        spatial_dims: int = 2,
+        is_complex: bool = True,
+        seed: Optional[int] = None,
+        allow_missing_keys: bool = False,
+    ) -> None:
+        super().__init__(keys, allow_missing_keys)
+        self.masker = RandomKspaceMask(
+            center_fractions=center_fractions,
+            accelerations=accelerations,
+            spatial_dims=spatial_dims,
+            is_complex=is_complex,
+            seed=seed,
+        )
+
+    def __call__(self, data: Mapping[Hashable, NdarrayOrTensor]) -> Dict[Hashable, Tensor]:
+        """
+        Args:
+            data: is a dictionary containing (key,value) pairs from the
+                loaded dataset
+
+        Returns:
+            the new data dictionary
+        """
+        d = dict(data)
+        for key in self.key_iterator(d):  # key is typically just "kspace"
+            d[key + "_masked"], d[key + "_masked_ifft"] = self.masker(d[key])
+            d["mask"] = self.masker.mask  # type: ignore
+            meta = "_meta_dict"
+            d["target"] = d[key + meta]["reconstruction_rss"]  # type: ignore
+            d["filename"] = d[key + meta]["filename"]  # type: ignore
```

Comment:
Could you please help define all these hard-code keys in Enum or somewhere?
Refer to: https://github.com/Project-MONAI/MONAI/blob/dev/monai/utils/enums.py#L406

Thanks.